### PR TITLE
Enable auth

### PR DESCRIPTION
**Pull request by techfort**

### CODE REVIEW

File: cassandra.sh.tpl [hunk 1]
```diff
@@ -35,13 +35,16 @@ apt-get update
 apt-get install -y gcc libev4 libev-dev python-dev
 apt-get install -y dsc30 -V
 apt-get install -y cassandra-tools
-service cassandra stop
+# service cassandra stop
```
Comment (lmammino) [inline cassandra.sh.tpl]: Why is this line commented?

Comment (techfort) [inline cassandra.sh.tpl]: ha - no reason other than a mistake

**Comment by techfort** [comment]
that's done, the line is no longer necessary

**Comment by lmammino** [comment]
you can proceed with the merge for me :)

**Pull request merged**